syftlib/src/main/java/org/openmined/syft/proto/SyftModel.kt

```diff
@@ -28,13 +28,16 @@ data class SyftModel(
 //        } ?: return null
 //    }
 
+    // Save/Update SyftModel
```

Comment:
Maybe add a note of how this function is used or just refer to the usage in demo app.

additionally, a more elaborate description would be better like call this function after every gradient step.